### PR TITLE
Document steps to install Mattermost Desktop from Repository

### PR DESCRIPTION
**Pull request by pfltdv**
#### Summary
Desktop Debian packages are published into Debian repository. Documentation is updated to install Mattermost desktop from our repository.

#### Ticket Link
JIRA: https://mattermost.atlassian.net/browse/DOPS-794

#### Preview Link
http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/5636/install/desktop-app-install.html#ubuntu-and-debian-based-systems

### CODE REVIEW

**Review by spirosoik — APPROVED** [review]

**Review by devinbinnie — APPROVED** [review]

File: source/install/desktop-app-install.rst [hunk 1]
```diff
@@ -101,19 +101,39 @@ Generic Linux package
...
+
+      - 64-bit systems ``sudo dpkg -i mattermost-desktop-5.0.4-linux-amd64.deb``
+      - 32-bit systems ``sudo dpkg -i mattermost-desktop-5.0.4-linux-i386.deb``
+
+    3. Run Mattermost as a desktop app.
+
+
+  .. tab:: Install from Mattermost Repository
+
+    Community-driven ``.deb`` packages are published into Mattermost Repository.
```
Comment (cwarnermm) [inline source/install/desktop-app-install.rst:125]:
```suggestion
    Community-driven ``.deb`` packages are published in the Mattermost repository.
```

**Review by cwarnermm — APPROVED** [review]
Thank you, @pfltdv! I've added this PR to our v6.7-documentation branch, and I've provided feedback inline.

**Comment by pfltdv** [comment]
> Thank you, @pfltdv! I've added this PR to our v6.7-documentation branch, and I've provided feedback inline.

Thanks alot @cwarnermm, changes are applied to the PR.

**Comment by cwarnermm** [comment]
@pfltdv - After a discussion with @nab-77, we've confirmed that the updates in this PR have been captured in https://github.com/mattermost/docs/pull/5628 (where the bulk of the Desktop App install, deploy, and update details live, including the new automatic updates functionality coming in v5.1.)

Given that this PR is a duplicate, I'm going to close it. I really appreciate you creating this PR.

**Pull request closed without merging**